src/libraries/Microsoft.Extensions.Configuration.Binder/tests/ConfigurationCollectionBindingTests.cs

```diff
@@ -211,6 +211,69 @@ public void GetUintEnumDictionary()
             Assert.Equal("val_3", options[KeyUintEnum.ghi]);
         }
 
+        [Fact]
+        public void GetSByteDictionary()
+        {
+            GetIntDictionaryT<sbyte>(0, 1, 2);
+        }
+        [Fact]
+        public void GetByteDictionary()
+        {
+            GetIntDictionaryT<byte>(0, 1, 2);
+        }
+        [Fact]
+        public void GetShortDictionary()
+        {
+            GetIntDictionaryT<short>(0, 1, 2);
+        }
+        [Fact]
+        public void GetUShortDictionary()
+        {
+            GetIntDictionaryT<ushort>(0, 1, 2);
+        }
+        [Fact]
+        public void GetIntDictionary()
+        {
+            GetIntDictionaryT<int>(0, 1, 2);
+        }
+        [Fact]
```

Comment:
nit: keep blank line above each `[Fact]`

```suggestion
        }

        [Fact]
```